### PR TITLE
Change favicons

### PR DESCRIPTION
**Pull request by not7cd**

### CODE REVIEW

File: favicon/site.webmanifest [hunk 1]
```diff
@@ -1,19 +1,19 @@
 {
-    "name": "Hackerspace Pomorze",
-    "short_name": "Hackerspace",
+    "name": "hsp.sh",
+    "short_name": "hsp.sh",
     "icons": [
         {
-            "src": "/favicon/android-chrome-192x192.png",
+            "src": "android-chrome-192x192.png",
```
Comment (yasiupl) [inline favicon/site.webmanifest:6]: This resource doesn't exist. add /favicon/ to the src.

File: favicon/site.webmanifest [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
             "sizes": "192x192",
             "type": "image/png"
         },
         {
-            "src": "/favicon/android-chrome-384x384.png",
-            "sizes": "384x384",
+            "src": "android-chrome-256x256.png",
```
Comment (yasiupl) [inline favicon/site.webmanifest:11]: The same applies here.

File: _layouts/meta.html [hunk 3]
```diff
@@ -7,13 +7,16 @@
...
-    <link rel="icon" type="image/png" sizes="16x16" href="/favicon/pixel-hsp.png">
-    <link rel="apple-touch-icon" sizes="180x180" href="/favicon/apple-touch-icon.png">
-    <link rel="mask-icon" href="/favicon/safari-pinned-tab.svg" color="#737373">
 
-    <meta name="msapplication-TileColor" content="#2b5797">
-    <meta name="theme-color" content="#000000">
+    <link rel="apple-touch-icon" sizes="180x180" href="/favicon/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="/favicon/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="/favicon/favicon-16x16.png">
+    <link rel="manifest" href="/favicon/site.webmanifest">
```
Comment (yasiupl) [inline _layouts/meta.html:14]: Now we have two manifest files. Remove one and edit the other to reflect the changes.

**Review by yasiupl — CHANGES_REQUESTED** [review]

**Comment by not7cd** [comment]
good to go

**Comment by yasiupl** [comment]
Cool, Now do DNS

**Pull request merged**